pkg/commands/patch/patch_manager.go

```diff
@@ -151,7 +154,7 @@ func (p *PatchManager) RemoveFileLineRange(filename string, firstLineIdx, lastLi
 		return err
 	}
 	info.mode = PART
-	info.includedLineIndices = utils.DifferenceInt(info.includedLineIndices, getIndicesForRange(firstLineIdx, lastLineIdx))
+	info.includedLineIndices, _ = lo.Difference(info.includedLineIndices, getIndicesForRange(firstLineIdx, lastLineIdx))
```

Comment:
lo returns a second argument giving the difference of B-A. Not sure how I feel about that, I've rarely needed it.